mmv1/products/compute/Subnetwork.yaml

```diff
@@ -203,6 +206,10 @@ properties:
     custom_expand: 'templates/terraform/custom_expand/resourceref_with_validation.go.tmpl'
     resource: 'Network'
     imports: 'selfLink'
+  - name: 'numericId'
+    type: String
```

Comment:
It's an integer in the API, but followed the example from `google_compute_network` of making it a string, so I assume that's preferred?